test/functional/features/ask-for-more-time.feature

```diff
@@ -45,3 +45,90 @@ Feature: Ask for more time page
     Then I see Your request for more time has been sent screen
     When I click "See your appeal progress" button
     Then I am on the overview page
+
+  Scenario: AFMT without evidence
```

Comment:
Scenarios have the same name/description, can we add a more descriptive name please?